goagen/gen_app/test_generator.go

```diff
@@ -58,7 +58,11 @@ func (g *Generator) generateResourceTest() error {
 	if len(g.API.Resources) == 0 {
 		return nil
 	}
-	funcs := template.FuncMap{"isSlice": isSlice}
+	funcs := template.FuncMap{
+		"isSlice":       isSlice,
+		"initArguments": initArguments,
```

Comment:
Does `initArguments` need to be called when the template is rendered? It could be instead called by the code prior to rendering the template which would be better.